src/main/java/org/ice4j/TransportAddress.java

```diff
@@ -181,6 +199,19 @@ public String getHostAddress()
         return addressStr;
     }
 
+    /* Return the host address, redacted if address redaction is enabled. */
+    public String getRedactedAddress()
+    {
+        if (AgentConfig.config.getRedactRemoteAddresses())
+        {
+            return toRedactedString(getAddress());
+        }
+        else
+        {
+            return getHostAddress();
```

Comment:
This is missing the fallback to `getHostName()`, so changes behavior (though I don't understand the fallback).